test/test_exif2timestream.py

```diff
@@ -9,6 +9,16 @@
 import unittest
 from voluptuous import MultipleInvalid
 from tempfile import NamedTemporaryFile
+import pexif
+import warnings
+SKIMAGE = False
+try:
+    from skimage.transform import resize
+    import skimage.io as io
+    from skimage import novice
```

Comment:
The comment I made [here](https://github.com/borevitzlab/exif2timestream/pull/22/files#r32087099) applies in this section too.